jbpm-human-task/jbpm-human-task-audit/src/test/java/org/jbpm/services/task/audit/service/TaskAuditBaseTest.java

```diff
@@ -1328,6 +1327,33 @@ public void testUpdateTaskContentEvents() {
         Assertions.assertThat(taskEvents.get(5).getType()).isEqualTo(TaskEventType.COMPLETED);
         Assertions.assertThat(taskEvents.get(5).getUserId()).isEqualTo("Darth Vader");
     }
+    
+    @Test
+    public void testForwardTaskWithMsgEvents() {
+        Task task = new TaskFluent().setName("This is my task name")
+                .addPotentialGroup("Knights Templer")
+                .setAdminUser("Administrator")
+                .getTask();
+
+        taskService.addTask(task, new HashMap<String, Object>());
+        long taskId = task.getId();
+
+        assertAuditTaskInfoGroup("Knights Templer", "Ready", taskId);
+
+        taskService.forward(taskId, "Administrator", "Jon Snow");
```

Comment:
I do not see user with name "Jon Snow" defined anywhere. I think it is the reason the test fails

Reply:
Done